pkg/route/api/validation/validation.go

```diff
@@ -300,9 +301,14 @@ func validateCertificatePEM(certPEM string, options *x509.VerifyOptions) (*x509.
 	}
```

Comment:
not sure what to do if there are non-CERTIFICATE PEM blocks in here... will that choke the router?